subproviders/nonce-tracker.js

```diff
@@ -59,7 +59,7 @@ NonceTrackerSubprovider.prototype.handleRequest = function(payload, next, end){
         var rawTx = payload.params[0]
         var stripped = ethUtil.stripHexPrefix(rawTx)
         var rawData = Buffer.from(ethUtil.stripHexPrefix(rawTx), 'hex')
-        var tx = new Transaction(Buffer.from(ethUtil.stripHexPrefix(rawTx), 'hex'))
+        var tx = TransactionFactory.fromSerializedData(Buffer.from(ethUtil.stripHexPrefix(rawTx), 'hex'))
```

Comment:
Do we need to pass through the chain options here (hardfork, chain, etc.)? I'm not sure if it matters...